### DISCUSSION TITLE
[M-Z] (15 Gennaio 2018) Esercizio 5

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#427] [Esami & Esoneri M-Z]
## Traccia

![image](https://github.com/user-attachments/assets/cb162e64-2476-46e2-bfa4-c2f0d928e3de)

> Questo esercizio è stato proposto **anche** come **Esercizio 5** della traccia di esame **del 22 gennaio 2019**

**Comment by rimaout** [comment]
![image](https://github.com/user-attachments/assets/fa160152-8b63-4c7d-9124-7d03748bfa00)

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [*20190122.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20190122.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/1165edbb-cdcc-45fc-8ce2-dd2102a76b5f)

\* _Questo esercizio era in comune rispetto alla traccia del 22 gennaio 2019, gli esercizi 7, 8 e 9 sono specifici delle due rispettive tracce, per quelli vedi [20180115.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20180115.pdf)_
